heroku_connect/models.py

```diff
@@ -289,3 +301,23 @@ class Meta(TriggerLogAbstract.Meta):
         db_table = '{schema}"."_trigger_log_archive'.format(schema=settings.HEROKU_CONNECT_SCHEMA)
         verbose_name = _('Trigger Log (archived)')
         verbose_name_plural = _('Trigger Logs (archived)')
+
+    @transaction.atomic
+    def redo(self):
+        """
+        Re-apply the change recorded in this trigger log.
+
+        .. seealso:: :meth:`.TriggerLog.redo`
+        """
+        trigger_log = self._copy_to_trigger_log(state=TRIGGER_LOG_STATE['NEW'])
+        trigger_log.save()
+        self.state = TRIGGER_LOG_STATE['REQUEUED']
+        self.save(update_fields=['state'])
+        return trigger_log
+
+    def _copy_to_trigger_log(self, **kwargs):
```

Comment:
Why not just use `copy`?
If copy doesn't work, you should override `__copy__` on the model class.

Reply:
Another case of lacking documentation on my end. The data of gets copied into an instance of a different class (`TriggerLogArchive` -> `TriggerLog`). I'll make that clearer.

Reply:
how about a different name `to_trigger_log` and add a docstring ;)

Reply:
That's exactly what I'm doing at the moment. :)